pkg/provider/kubernetes/ingress/kubernetes.go

```diff
@@ -335,10 +335,7 @@ func (p *Provider) loadConfigurationFromIngresses(ctx context.Context, client Cl
 					rules = append(rules, "PathPrefix(`"+p.Path+"`)")
 				}
 
-				routerKey := strings.Replace(rule.Host, ".", "-", -1) + strings.Replace(p.Path, "/", "-", 1)
```

Comment:
the second strings.Replace was doing the replacement only on the first occurrence found, so is it safe to replace it with a call to provider.Normalize? is that what we want for sure?

Reply:
yes it's the expected behavior.